docs/Collaboration guidelines.md

```diff
@@ -0,0 +1,51 @@
+Collaboration Guidelines
+===
+
+Be polite to each other
+---
+
+No one wants to wake up to a rude reply. Try to behave the best way you can, and show understanding when others are not. 
+
+When things escalate, try to use this document as a guideline on how to de-escalate them.
+
+Think of the project first
+---
+
+We're primarily here to make Reicast better.
+
+When discussing things in the tracker, forum or discord, try to think of the project first, and not of personal issues you might have with people.
+
+Strive for: Thinking "Constructive feedback is good because it makes contributors feel accepted, valued and safe. When people feel safe they are more comfortable explaining their opinion/viewpoint, and that helps make the project better"
+
+Avoid: Thinking "Emotionally abusing people is fun and makes my (the abuser's) day better"
+
+Avoid negativity, be realistic, give constructive feedback
+---
+
+Before replying, ask yourself the following questions.
+- How will the person I'm replying to feel after reading my comment?
+- Am I motivating them to get work done, or just spreading negativity?
+- Am I giving feedback that is valuable?
+
+Gracefully recover from missteps
+---
+
+We are all humans, and we have emotions and troubles, so often our conduct is not ideal. Here are some ideas on how to get things back on track.
+- When not thinking with a clear mind, take a break. Go for a walk, call a friend, stare at TV, smoke a cig, whatever works for you.
+- When in the wrong, apologize, and try to not repeat the offending behavior. Remember, an apology is sincere _only_ if it changes your behavior.
+- Defend your opinions, but be open to exploring other ideas, especially after a disagreement. "A strong opinion, weakly held"
+
+Remember, we're doing science
+---
+
+- Try to convince others based on scientific/rational merit, not personal opinion or emotion. 
+- Try to extract knowledge, even when others are behaving in a not-so-nice way. Remember: almost every viewpoint has some value.
+- Try to decompose your thinking process and explain it to others, identify leaps of rationality.
+- Try to understand other's people's thinking process, and how they decompose the ideas in their heads.
+- Communicate without taking things personally
+
+A good rule of thumb is: If you can't explain why something should be done in some way, you're probably not thinking very rationally.
+
+Bad example: C++ is good because I know it
+Good example: C++ lets me do this and that, because of this and that, and I thin kit can work well because of ...
```

Comment:
Typo on last line

think it can

Reply:
Also, it's also a good idea to write down what we can do with people who remain negative,  aggressive,  toxic.

Reply:
Isn't that how you get a member badge? 👅